etc/config/c++.amazon.properties

```diff
@@ -4910,6 +4910,15 @@ libs.qt.versions.652.path=/app/qt/include/QtCore
 libs.qt.versions.660.version=6.6.0
 libs.qt.versions.660.path=/app/qt/include/QtCore
 
+libs.quill.name=Quill
+libs.quill.description=C++ Low Latency Logging Library
+libs.quill.versions=trunk:611
+libs.quill.url=https://github.com/odygrd/quill
+libs.quill.versions.trunk.version=trunk
```

Comment:
I didn't see an install for trunk quill in your infra PR: this directory doesn't exist. You will need a `trunk` type install that is either `if: nightly` and `method: nightlyclone` or is in the big block of targets like `benchmark` at the end of the library file that are all behind an `if:nightly` already, and then ensure it has a `nightlyclone` too. I can merge this with the proviso the quill trunk won't be ready yet.

Sorry it's not obvious: we try and keep the "nightly" and non-nightly separate for...now questionable reasons, but at one point it (sort of) made sense.